tests/functional/distributable.js

```diff
@@ -0,0 +1,13 @@
+const { expect } = chai;
+
+describe('Distributable', function suite() {
```

Comment:
How it’s different from the wallet functional test?

Reply:
Removed.